src/main/java/cms/gov/madie/measure/services/BundleService.java

```diff
@@ -64,14 +64,15 @@ public String bundleMeasure(Measure measure, String accessToken, String bundleTy
     throw new BundleOperationException("Measure", measure.getId(), null);
   }
 
-  public PackageDto getMeasureExport(Measure measure, String accessToken) {
+  public PackageDto getMeasureExport(
+      Measure measure, boolean includeElmWarnings, String accessToken) {
     if (measure == null) {
       return null;
     }
     if (measure.getMeasureMetaData().isDraft()) {
       return getMeasureExportForDraft(measure, accessToken);
```

Comment:
we have to do it for QiCore drafts as well

Reply:
Will be completed as part of MAT-8176.